turbo.json

```diff
@@ -11,7 +11,7 @@
     "build": {
       "dependsOn": ["pre-build", "^build"],
       "outputs": [],
-      "cache": true
+      "cache": false
```

Comment:
Annoyingly, having the cache turned on for builds seems to produce inconsistent results. Might be best to turn it off.